### DISCUSSION TITLE
7.18.2 not available

### DISCUSSION THREAD

**Discussion by kiransnair** [babel/babel#14598] [Releases] [answered]
Version     Downloads (Last 7 Days)         Tag     
[7.18.2]        0                                         latest
if this version is not available why it is listed the released list?
Application failing to download the latest packages 7.18.2 but that is not publicly available.

**Comment by nicolo-ribaudo** [comment, marked as answer]
It is publicly available: https://www.npmjs.com/package/@babel/types/v/7.18.2

**Reply by pecliu** [reply]
7.18.3 is not available
the latest version on npm is 7.18.2

**Reply by nicolo-ribaudo** [reply]
Of which package? We only publish a new version of a package when it contains some changes.
